modules/ml/src/main/java/org/apache/ignite/ml/environment/DefaultLearningEnvironmentBuilder.java

```diff
@@ -114,7 +130,7 @@ private static ParallelismStrategy strategyByType(ParallelismStrategy.Type stgyT
     @Override public LearningEnvironment buildForWorker(int part) {
         Random random = rngSupplier.apply(part);
         random.setSeed(seed.apply(part));
-        return new LearningEnvironmentImpl(part, random, parallelismStgy.apply(part), loggingFactory.apply(part));
+        return new LearningEnvironmentImpl(part, dataTtl, random, parallelismStgy.apply(part),  loggingFactory.apply(part));
```

Comment:
So long so large the constructor of LEI, maybe we should refactor to setters

Reply:
We introduced `LearningEnvironmentBuilder` to eliminate this problem. This constructor isn't in client API, it's used only in builder.